packages/parser/src/schemas/kinesis.ts

```diff
@@ -70,41 +70,48 @@ const KinesisDynamoDBStreamSchema = z.object({
  *         "partitionKey": "1",
  *         "sequenceNumber": "49590338271490256608559692538361571095921575989136588898",
  *         "data": "SGVsbG8sIHRoaXMgaXMgYSB0ZXN0Lg==",
- *         "approximateArrivalTimestamp": 1545084650.987
+ *         "approximateArrivalTimestamp": 1607497475.000
  *       },
  *       "eventSource": "aws:kinesis",
  *       "eventVersion": "1.0",
  *       "eventID": "shardId-000000000006:49590338271490256608559692538361571095921575989136588898",
  *       "eventName": "aws:kinesis:record",
- *       "invokeIdentityArn": "arn:aws:iam::123456789012:role/lambda-role",
- *       "awsRegion": "us-east-2",
- *       "eventSourceARN": "arn:aws:kinesis:us-east-2:123456789012:stream/lambda-stream"
- *     },
- *     {
- *       "kinesis": {
- *         "kinesisSchemaVersion": "1.0",
- *         "partitionKey": "1",
- *         "sequenceNumber": "49590338271490256608559692540925702759324208523137515618",
- *         "data": "VGhpcyBpcyBvbmx5IGEgdGVzdC4=",
- *         "approximateArrivalTimestamp": 1545084711.166
- *       },
- *       "eventSource": "aws:kinesis",
- *       "eventVersion": "1.0",
- *       "eventID": "shardId-000000000006:49590338271490256608559692540925702759324208523137515618",
- *       "eventName": "aws:kinesis:record",
- *       "invokeIdentityArn": "arn:aws:iam::123456789012:role/lambda-role",
- *       "awsRegion": "us-east-2",
- *       "eventSourceARN": "arn:aws:kinesis:us-east-2:123456789012:stream/lambda-stream"
```

Comment:
Why do we remove these fields from the example? Or was it unintentional?

Reply:
Hi Brice, now that we have the blog do you by chance have any PMM or adjacent function in your team that we could use to drive a bit of awareness for this integration?

Reply:
The example had two records in the array, I removed one for brevity.